ArduCopter/Copter.cpp

```diff
@@ -118,6 +118,10 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
 #if AC_CUSTOMCONTROL_MULTI_ENABLED
     FAST_TASK(run_custom_controller),
 #endif
+#ifdef SIMULINK_APP_ENABLED
+    // run Simulink-generated application
+    FAST_TASK(run_simulink_step),
```

Comment:
this looks like both controllers will run, it won't obey the AUX switch 106 CUSTOM_CONTROLLER, and will interact very badly with the rate thread work
I really think this needs to use the custom controller APIs